core/controllers/admin.py

```diff
@@ -635,6 +636,168 @@ def get(self):
         self.render_json(response)
 
 
+class AddCommunityReviewerHandler(base.BaseHandler):
+    """Handles adding reviewer for community dashboard page."""
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    @acl_decorators.can_access_admin_page
+    def post(self):
+        new_reviewer_username = self.payload.get('username')
+        new_reviewer_user_id = (
+            user_services.get_user_id_from_username(new_reviewer_username))
+
+        if new_reviewer_user_id is None:
+            raise self.InvalidInputException(
+                'Invalid username: %s' % new_reviewer_username)
+
+        review_category = self.payload.get('review_category')
```

Comment:
If no review_category is provided, what happens? Please ensure there is a test for this.

In general, take care to validate handlers very strictly; data coming from outside can be anything.

Reply:
> If no review_category is provided, what happens?

It will be None by default and will fail at line 689.

> Please ensure there is a test for this.

We have a test for invalid review_category in admin_test.py at line 1283. I'll add a test for no "review_category"